modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/operation/OperationMessageProcessorObjectFactory.java

```diff
@@ -62,13 +67,14 @@ public OperationMessageProcessor getObject() throws Exception {
   private OperationMessageProcessor createMessageProcessor(ResolverSet resolverSet) {
     if (operationModel.getModelProperty(InterceptingModelProperty.class).isPresent()) {
       return new InterceptingOperationMessageProcessor(extensionModel, operationModel, configurationProvider, target,
-                                                       resolverSet, (ExtensionManagerAdapter) muleContext.getExtensionManager());
+                                                       resolverSet, (ExtensionManagerAdapter) muleContext.getExtensionManager(),
+                                                       policyManager);
     } else if (operationModel.getModelProperty(PagedOperationModelProperty.class).isPresent()) {
       return new PagedOperationMessageProcessor(extensionModel, operationModel, configurationProvider, target, resolverSet,
```

Comment:
Even though they do, all their configuration is set through constructor. I think if better if we keep it consistent.